wlm/src/main/java/ai/djl/serving/wlm/PyAdapter.java

```diff
@@ -35,33 +32,23 @@ protected PyAdapter(String name, String src, Map<String, String> options) {
 
     @SuppressWarnings("unchecked")
     @Override
-    protected void registerPredictor(Predictor<?, ?> predictor) {
-        Predictor<Input, Output> p = (Predictor<Input, Output>) predictor;
+    protected <I> I getRegisterInput() {
```

Comment:
same as above, can we change to `getRegisterAdapterInput` and `getUnregisterAdapterInput`

Reply:
Changed.